sigs.yaml

```diff
@@ -2144,3 +2144,30 @@ workinggroups:
       tz: "PT (Pacific Time)"
       frequency: weekly
       url: https://docs.google.com/document/d/1RbC4SBZBlKth7IjYv_NaEpnmLGwMJ0ElpUOmsG-bdRA/edit
+  - name: K8s Infra
+    dir: wg-k8s-infra
+    mission_statement: >
+      A Working Group dedicated to migrating Kubernetes project infrastructure
+      over to the CNCF, and the creation of teams and processes for ongoing
+      maintenance. Involves collaboration with multiple SIGs such as
+      Architecture, Contributor Experience, Release, and Testing.
```

Comment:
Testing, etc.

Reply:
Is there any etc here? I'd like to eventually have working groups list out the sigs they're tied to, so if this list is incomplete I'd like to fill it out.  I think there will be downstream consumer SIGs like cluster-lifecycle, I tried to keep this list to sigs that seem to canonically "own" these infra today.

Reply:
Have added the etc, will want to revisit when we quantify list of SIGs (ref: https://github.com/kubernetes/community/issues/2176)